### ISSUE TITLE
Don't skip `link:` in `elfeed-show-next-link`

### ISSUE THREAD

**Issue by ParetoOptimalDev** [skeeto/elfeed#482]
For instance in:

![image](https://user-images.githubusercontent.com/93023868/223572767-b896f863-57cb-463a-9565-f5bef0e64f81.png)

I'd like hitting TAB to go to `Link: https://news.ycombinator.com/item?id=35062370`.

It looks like a simple change if it's universally agreeable, or even agreeable behind a config flag with:

```diff
(defun elfeed-show-next-link ()
  "Skip to the next link, exclusive of the Link header."
  (interactive)
  (let ((properties (text-properties-at (line-beginning-position))))
    (when (memq 'message-header-name properties)
+    (previous-line)
      (forward-paragraph))
    (shr-next-link)))
```

**Comment by plantarum** [comment]
I had the same idea. After some investigating, I assume the reason this isn't the default behaviour is that you can already open the link for an entry with `b`, bound to `elfeed-show-visit`.

**Comment by minad** [comment]
Yes, this is intentional. Press b or B if you want to open the Link: url.
